helical/models/hyena_dna/fine_tuning_model.py

```diff
@@ -0,0 +1,168 @@
+from typing import Literal, Optional
+from helical.models.base_models import HelicalBaseFineTuningHead, HelicalBaseFineTuningModel, HelicalDNAModel
+from torch import optim
+import torch
+from torch.nn.modules import loss
+from .hyena_dna_utils import HyenaDNADataset
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+from transformers import get_scheduler
+import logging
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+class HyenaDNAFineTuningModel(HelicalBaseFineTuningModel):
+    """HyenaDNA fine-tuning model.
+
+    This class represents the HyenaDNA fine-tuning model, which is a long-range genomic foundation model pretrained on context lengths of up to 1 million tokens at single nucleotide resolution.
+
+    Parameters
+    ----------
+    hyena_model : HelicalDNAModel
```

Comment:
We know this is of HyenaDNA type right?
I think you can make this change to GeneformerFuneTuningModel etc also